data/packages.yml

```
@@ -127,7 +141,10 @@
   date_added: 2024-09-21
   last_updated: 2024-09-21
   url: https://github.com/gwkokab/gwkokab
-  authors: Meesum Qazalbash, Muhammad Zeeshan, Richard O'Shaughnessy
+  authors:
+    - name: Meesum Qazalbash
+    - name: Muhammad Zeeshan
+    - name: Richard O'Shaughnessy
```

Can you update this entry?

```suggestion
  authors:
    - name: Meesum Qazalbash
      url: https://github.com/Qazalbash
    - name: Muhammad Zeeshan
      url: https://ccrg.rit.edu/user/muhammad.zeeshan
    - name: Richard O'Shaughnessy
      url: https://ccrgpages.rit.edu/~oshaughn/Richard_OShaughnessy/Home.html
```

---

Absolutely